src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1866,19 +1900,19 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
         $container->getDefinition('messenger.retry_strategy_locator')
             ->replaceArgument(0, $transportRetryReferences);
 
-        if ($config['failure_transport']) {
-            if (!isset($senderReferences[$config['failure_transport']])) {
-                throw new LogicException(sprintf('Invalid Messenger configuration: the failure transport "%s" is not a valid transport or service id.', $config['failure_transport']));
-            }
+        if (\count($failureTransports) > 0) {
+            $globalFailureReceiverName = $config['failure_transport'] ?? null;
```

Comment:
I think (?) based on old code that this will always be set or `null`? So maybe the `??` isn't needed? And maybe we could just use `$config['failure_transport']` directly like before?